app/scenes/Settings/NocoDb.tsx

```diff
@@ -0,0 +1,101 @@
+import { observer } from "mobx-react";
+import { TableIcon } from "outline-icons";
+import * as React from "react";
+import { useForm } from "react-hook-form";
+import { useTranslation, Trans } from "react-i18next";
+import { IntegrationType } from "@shared/types";
+import Integration from "~/models/Integration";
+import Button from "~/components/Button";
+import Heading from "~/components/Heading";
+import { ReactHookWrappedInput as Input } from "~/components/Input";
+import Scene from "~/components/Scene";
+import Text from "~/components/Text";
+import useStores from "~/hooks/useStores";
+import useToasts from "~/hooks/useToasts";
+
+type FormData = {
+  url: string;
+};
+
+const SERVICE_NAME = "nocodb";
+
+function NocoDb() {
```

Comment:
Appreciate you adding this page, I think that in hindsight these individual pages will quickly get out of control – we also have one for TLDraw in the wings. It's probably better if we just rename the existing one for "Draw.io" to be "Self hosted" and then add the configuration for all self-hosted integrations in that one page under sub-headings, thoughts?

Reply:
Yep that would make sense !

Reply:
@Limezy would you be interested in giving me a hand with this? Still working on another PR for Outline I need asap, hard to find time for everything

Reply:
Yep will have a try

Reply:
Thanks @tommoor for having started the work ! Didn't find any time yet